app/lib/constants.js

```diff
@@ -29,7 +46,7 @@ module.exports = {
     nearby: 'nearby',
     openNearby: 'openNearby',
   },
-  siteRoot: '/find-a-pharmacy',
+  siteRoot: '/service-search/find-a-pharmacy',
```

Comment:
This should be the only change in this file. The other changes should be reverted.

Reply:
I have reverted the linting issues now

Reply:
I'm still seeing the changes where the days of week are on separate lines (https://github.com/nhsuk/connecting-to-services/pull/617/files#diff-8216c7e4b1f6e7b1a8d5058460194e55R23) and `description` has still got `"` rather than `'`.